### PR TITLE
TI-2943 - Add Appurtenances to Main Schema for Sewer Module

### PR DESCRIPTION
**Pull request by stewartm83**
Please review

### CODE REVIEW

File: schema/main.schema [hunk 1]
```diff
@@ -2341,6 +2341,49 @@ CREATE SUFFIX TABLES
 	Future, Existing;
 };
 
+CREATE TABLE "" SewerAuxAppurtenance : SewerBaseNoID
+{
+	"Internal"
+	optional  point             	Geometry                                        ""
+	optional  double              	ROTATION                                        ""
+	optional  uuid		           	id				                                "ID"
```
Comment (AnthonyThevenin) [inline schema/main.schema]: Is it `id` or `ID`?

File: schema/main.schema [hunk 2]
```diff
@@ -2341,6 +2341,49 @@ CREATE SUFFIX TABLES
...
+	optional  text		           	SYSTEM                                          "System"															uiorder:11
+	optional  text		           	COMMENT                                         "Comment"															uiorder:12
+	optional  text		           	REGION                                          "Region"															uiorder:13
+	optional  text		           	NETWORK                                         "Drainage Area"														uiorder:14
+	"04.Asset Information"
+	optional  text		           	S_MAKE                                          "Make"																uiorder:15
+	"05.Integrity"
+	optional  text		           	S_YR_INT                                        "Year Integrity"													uiorder:16
+	optional  text		           	S_DRW_NO                                        "Drawing No"														uiorder:17
+	"Internal"
```
Comment (AnthonyThevenin) [inline schema/main.schema]: remove that

**Comment by AnthonyThevenin** [comment]
Sorry there are already `uiorder`!!

**Comment by stewartm83** [comment]
Okay. Will remove that **Internal** line.

**Comment by AnthonyThevenin** [comment]
Ok. You should also improve the `uiorder`. They should start at `0` to `n-1` for each groups. And don't use `01 02 ..`.
Also, since you have an `ID`, you should use `SewerBase` instead of `SewerBaseNoID`.

**Pull request merged**